external-crates/move/crates/move-compiler/tests/move_2024/typing/all_dot_paths_valid.move

```diff
@@ -1,6 +1,6 @@
 module a::m {
     const ZED: Z = Z { f: 0 };
-    const V: vector<u64> = vector[0];
+    const VEC: vector<u64> = vector[0];
```

Comment:
What was this change for? Not opposed to it, just making sure I understand

Reply:
My sanity to avoid swapping `Z` and `V` on accident